test/built-ins/Object/getOwnPropertyDescriptors/duplicate-keys.js

```diff
@@ -27,5 +27,5 @@ var proxy = new Proxy({}, {
 
 var result = Object.getOwnPropertyDescriptors(proxy);
 assert.sameValue(result.hasOwnProperty('DUPLICATE'), true);
-assert.sameValue(result.DUPLICATE, undefined);
+assert.sameValue(result.DUPLICATE, descriptors[descriptors.length - 1]);
 assert.sameValue(log, 'ownKeys|getOwnPropertyDescriptor:DUPLICATE|getOwnPropertyDescriptor:DUPLICATE|getOwnPropertyDescriptor:DUPLICATE|');
```

Comment:
Last nit: the result from the trap is not the same value as the item added to the result object, even though in this case they should have the same keys and values.

https://tc39.github.io/ecma262/#sec-proxy-object-internal-methods-and-internal-slots-getownproperty-p